modules/serverless-installing-knative-serving.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+[id="installing-knative-serving_{context}"]
+= Installing Knative Serving
+
+You need to create a `KnativeServing` object to install Knative Serving using the Openshift Serverless Operator.
+
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference. 
```

Comment:
What does this tell me as a user?

Reply:
Should probably be "This sample yaml shows the minimum configuration to create a KnativeServing object:" or something like that